Documents/flashing

```diff
@@ -6,17 +6,21 @@
 
 Visually identify any content that flashes. Flashing is content that rapidly alternates between two or more states that vary significantly in contrast.
 
-When flashing content IS found, the result for Test ID 3.A <span id="OLE_LINK17" class="anchor"></span>(Test Name: 2.3.1-flashing) is **NOT TESTED**.
+-	  It is recommended that testers inquire about occurrences of flashing content. Vendors/developers should make testers aware of the location of any flashing content. (does this belong in this doc? Or e.g. guidance including sampling guide)
```

Comment:
I think it makes good sense for the guidance doc instead of the test process.

Reply:
As a tester I would prefer a "one-stop shopping" solution. Maybe add a guidance section when needed per Test ID?

Reply:
Is there already a tester's guide to the galaxy? It would include questions to ask and what testers need to prepare for testing. The Sampling guidance would be good there also.